src/shared/enums.ts

```diff
@@ -116,14 +118,27 @@ export enum AutomationRegistrarSubtask {
   getTypeAndVersion = "getTypeAndVersion",
 }
 
+export enum FunctionsSubtask {
```

Comment:
for consistency with other objects in the enums, we should rename this to `FunctionsSubscriptionSubtasks`